src/pal/src/misc/perftrace.cpp

```diff
@@ -33,7 +33,7 @@ Module Name:
 #include <string.h>
 #include <errno.h>
 
-#define snprintf _snprintf
+#define snprintf _snprintf_s
```

Comment:
@jkotas thanks for spotting this. Yes, I was going to change it and then decided I will remove this file later since it is of no future use for CoreCLR, but forgotten to revert the change here.